lighthouse-cli/test/check-formatting.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
```

Comment:
can you move this to node?
adding a 3rd language for utility scripts does not sound great.